src/SFA.DAS.EmployerIncentives.Data/Apprenticeship/UlnQueryRepository.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using SFA.DAS.EmployerIncentives.Data.Models;
+using SFA.DAS.EmployerIncentives.Enums;
+
+namespace SFA.DAS.EmployerIncentives.Data.Apprenticeship
+{
+    public class UlnQueryRepository : IUlnQueryRepository
```

Comment:
I don't think we should be using QueryRepos for domain logic. In reality, this probably just means renaming this class.

Reply:
Happy to call it something else. Just not sure what.

Reply:
renamed to UlnValidationServce